doc/website/getting-started/overview.md

```diff
@@ -0,0 +1,540 @@
+            });
+    ```
+    Notice that we get an untyped sample, ``iox::popo::Sample<void>`` (we could also use ``auto& sample`` in the
+    lambda arguments to shorten it). Again we access the pointer to the underlying raw memory of the sample and
+    construct the data we want to send.
+
+
+
+#### Creating a subscriber
+
+While the string identifiers still have to match those in the publisher, as in the untyped publisher there is no
+template type argument.
+
+```
+iox::popo::UntypedSubscriber subscriber({"Group", "Instance", "CounterTopic"});
+subscriber.subscribe();
+```
+#### Receiving data
+
+Receiving the data works in the same way as in the typed API, the main difference is the ``reinterpret_cast`` needed
+before accessing the data (since the subscriber has no knowledge of the underlying type).
+
+
+```
+while(keepRunning) 
+{
+    //wait for new data (either sleep and wake up periodically or by notification from the waitset)
+
+    subscriber->take()
+        .and_then([](iox::popo::Sample<const void>& sample) {
+            CounterTopic* ptr = reinterpret_cast<CounterTopic*>(sample.get());
+            /* process the received data using the ptr */         
+        })
+        .if_empty([] { /* no data received but also no error */ })
+        .or_else([](iox::popo::ChunkReceiveError) { /* handle the error */ });
+}
+```
+
+Note that since the received sample received is untyped (``iox::popo::Sample<const void>``), we cannot use
+``operator->`` to access the members of the underlying type but have to cast it to the correct type ``CounterTopic``
+manually.
+
+As in the untyped case we also could use a loop to get all samples as long as they are available.
+
+
+### Shutdown
+
+Once we are done sending, we call ``stopOffer`` at the publisher.
+
+```
+publisher.stopOffer();
+```
+
+Similarly the subscriber can ``unsubscribe`` to stop receiving any data.
+
+```
+subscriber.unsubscribe();
+```
+
+Both will also be called in the respective destructor if needed (i.e. if the publisher is still offering or the
+subscriber is still subscribed).
+
+### Running an iceoryx system
+
+Now that we have applications capable of sending and receiving data, we can run the complete iceoryx system.
+
+First we need to start Roudi.
+
+    # If installed and available in PATH environment variable
+    iox-roudi
+    # If build from scratch with script in tools
+    $ICEORYX_ROOT/build/posh/iox-roudi
+
+Afterwards we can start the applications which immediately connect to the Roudi via their runtime.
+
+When the applications terminates the runtime cleans up all resources needed for communication with Roudi. This
+includes all memory chunks used for the data transmission which may still be hold by the application.
+
+## Examples
+
+This covers the main use cases and should enable the user to quickly develop iceroyx applications.
+
+Full examples and instructions on how to build and run them can be found in
+[examples](../iceoryx_examples/README.md).
```

Comment:
Probably we should add prioritises to the examples e.g. start with icedelivery

Reply:
can do.